src/bin/travis/test.sh

```diff
@@ -3,19 +3,27 @@
 set -e
 
 SCRIPT_DIR=src/bin/travis
-
-# BUILD_TYPE is coming in from the Travis build matrix
-if [ "$BUILD_TYPE" = "centaurJes" ]; then
-    "${SCRIPT_DIR}"/testCentaurJes.sh
-elif [ "$BUILD_TYPE" = "centaurTes" ]; then
-    "${SCRIPT_DIR}"/testCentaurTes.sh
-elif [ "$BUILD_TYPE" = "centaurLocal" ]; then
-    "${SCRIPT_DIR}"/testCentaurLocal.sh
-elif [ "$BUILD_TYPE" = "sbt" ]; then
-    "${SCRIPT_DIR}"/testSbt.sh
-elif [ "$BUILD_TYPE" = "checkPublish" ]; then
-    "${SCRIPT_DIR}"/testCheckPublish.sh
+# $TRAVIS_EVENT_TYPE will be cron is this build was initiated by a cron job
+if [ "$TRAVIS_EVENT_TYPE" != "cron" ]; then
```

Comment:
is there something that can positively be tested for here rather than `!= "cron"`?